### PR TITLE
StaticHtmlReporter: Speed-up collecting the analyzer issues

### PR DESCRIPTION
**Pull request by fviernau**

### CODE REVIEW

File: reporter/src/main/kotlin/reporters/ReportTableModelMapper.kt [hunk 1]
```diff
@@ -94,6 +94,7 @@ class ReportTableModelMapper(private val resolutionProvider: ResolutionProvider)
             val allIds = sortedSetOf(project.id)
             allIds += project.collectDependencies()
 
+            val projectIssues = project.collectIssues()
```
Comment (sschuberth) [inline reporter/src/main/kotlin/reporters/ReportTableModelMapper.kt]: Commit message:
- "in order to [collect] the issues in O(n) i[ns]tead [of] O(n²) time"
- "due to a[n] unusual large amount [of] scopes"

"was observed to take about one hour" -> so, how fast is it now?

Comment (fviernau) [reply reporter/src/main/kotlin/reporters/ReportTableModelMapper.kt]: Done.

Comment (mnonnenmacher) [reply reporter/src/main/kotlin/reporters/ReportTableModelMapper.kt]: Also commit message:
* "Use Project.collectIssue**s**()"
* Still: "amount **of** scopes"

Comment (fviernau) [reply reporter/src/main/kotlin/reporters/ReportTableModelMapper.kt]: Done.

**Pull request merged**